themes/doks/layouts/shortcodes/glossary.html

```diff
@@ -0,0 +1,13 @@
+{{/* Function to display the full list of terms and definitions defined for Corda 5 in glossaryitems.json. */}}
+{{/*The "definition" field in glossaryitems.json should only be one sentence. This is the text displayed as the ToolTip throughout the content and also displayed by default on the Glossary page.*/}}
+{{/*The "more" field in glossaryitems.json displays any additional content displayed by expanding the paragraph on the Glossary page.*/}}
+
+{{ $glossarylist := getJSON "content/en/platform/corda/5.0-beta/glossaryitems.json" }}
```

Comment:
I think that the next step is to generalize it.
In other words not be version specific ...probably the glossaryitems for Corda haven't changed in a while and won't change much going fwd ...
Then you'd have a master `glossaryitems.json` file and that would suit all versions of Corda.
@nadinequinr3 / @colin-hillis